src/collections/_documentation/learn/configuration.md

```diff
@@ -0,0 +1,204 @@
+
+{:.config-key}
+### sample-rate
+
+Configures the sample rate as a percentage of events to be sent in the range of `0.0` to `1.0`.  The
+default is `1.0` which means that 100% of events are sent.  If set to `0.1` only 10% of events will
+be sent.  Events are picked randomly.
+
+{:.config-key}
+### max-breadcrumbs
+
+This variable controls the total amount of breadcrumbs that should be captured.  This defaults
+to `100`.
+
+{:.config-key}
+### attach-stacktrace
+
+When enabled stacktraces are automatically attached to all messages logged.  Note that stacktraces
+are always attached to exceptions but when this is set stacktraces are also sent with messages.  This
+for instance means that stacktraces appear next to all log messages.
+
+It's important to note that grouping in Sentry is different for events with stacktraces and without.
+This means that you will get new groups as you enable or disable this flag for certain events.
+
+This feature is off by default.
+
+{:.config-key}
+### send-default-pii
+
+If this flag is enabled then certain personally identifyable information is added by active
+integrations.  Without this flag they are never added to the event to begin with.  If possible
+it's recommended to turn on this feature and use the server side PII stripping to remove the
+values instead.
+
+{:.config-key}
+### server-name
+
+{% unsupported browser %}
+Can be used to supply a "server name".  When provided the name of the server is sent along and
+persisted in the event.  Note that for many integrations the server name actually corresponds to
+the device hostname even in situations where the machine is not actually a server.  Most SDKs
+will attempt to auto discover this value.
+{% endunsupported %}
+
+{:.config-key}
+### blacklist-urls
+
+{% supported browser %}
+A pattern for error URLs which should not be sent to Sentry.  To whitelist certain errors instead,
+use `whitelist-urls`.  By default, all errors will be sent.  This only applies to events sent
+from browsers.
+{% endsupported %}
+
+{:.config-key}
+### whitelist-urls
+
+{% supported browser %}
+A pattern for error URLs which should exclusively be sent to Sentry.  To
+blacklist certain errors instead, use `blacklist-urls`.  By default, all errors
+will be sent.  This only applies to events sent from browsers.
+{% endsupported %}
+
+{:.config-key}
+### request-bodies
+
+{% supported python %}
+This parameter controls if integrations should capture HTTP request bodies.  It can be set to one
+of the following values:
+
+* `never`: request bodies are never sent.
+* `small`: only small request bodies will be captured where the cutoff for small depends on the SDK (typically 4KB)
+* `medium`: medium sized requests and small requests will be captured. (typically 10KB)
+* `always`: the SDK will always capture the request body for as long as sentry can make sense of it
+{% endsupported %}
+
+{:.config-key}
+### with-locals
```

Comment:
Does it make sense to default this to true while send-default-pii is defaulting to false?